src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskController.kt

```diff
@@ -80,6 +81,13 @@ class TaskController {
             endDate = task.endDate,
             milestone = milestone,
             status = TaskStatus.NOT_STARTED,
+                // Todo: match types below
+            assignees = emptyList(),
```

Comment:
create new TaskAssigneeEntity here, also create TaskAssigneeRepository to create those/list by task